src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1406,151 +1403,145 @@
   "com.minecolonies.coremod.info.lumberjack.1.name": "Restrictions",
-  "com.minecolonies.coremod.info.lumberjack.1": "Now let's cover restrictions:\n\n 1. Lumberjacks unlock certain tool levels per their hut level:\n   lvl 0. Wood/Gold Tools\n   lvl 1. Stone Tools\n   lvl 2. Iron Tools\n   lvl 4. Diamond Tools\n   lvl 5. Enchanted Tools\n\n 2. Lumberjacks are able to search for trees within a 150 blocks of their worker (not the hut), while searching they start with a 50 block search range and then increase that range by 5 when they do not find a tree.",
+  "com.minecolonies.coremod.info.lumberjack.1": "Now let's cover restrictions:\n\n 1. Lumberjacks unlock certain tool levels per their hut level:\n   lvl 0. Wood/Gold Tools\n   lvl 1. Stone Tools\n   lvl 2. Iron Tools\n   lvl 4. Diamond Tools\n   lvl 5. Enchanted Tools\n\n 2. Lumberjacks are able to search for trees within 150 blocks of them (not their hut). While searching, they start with a 50 block range and then increase that range by 5 if they do not find a tree.",
   "com.minecolonies.coremod.info.lumberjack.2.name": "Tree Detection",
-  "com.minecolonies.coremod.info.lumberjack.2": "3. The definition of a \"Tree\" in Minecraft is pretty hazy at best, we've done our best to define a tree, but in some cases it may not be enough. Generally we define a tree as a log and leaves (within a near vicinity), due to this some of your huts or decorations may be affected by the lumberjack, to counter this you can place a piece of Cobblestone below a log to mark it as \"not a tree\".",
+  "com.minecolonies.coremod.info.lumberjack.2": "3. The definition of a \"Tree\" in Minecraft is pretty hazy at best. We've done our best to define a tree, but in some cases it may not be enough. Generally we define a tree as a log and leaves (near each other), due to this some of your huts or decorations may be affected by the lumberjack, to counter this you can place a piece of Cobblestone below a log to mark it as \"not a tree\".",
   "com.minecolonies.coremod.info.lumberjack.3.name": "Leveling",
-  "com.minecolonies.coremod.info.lumberjack.3": "As all workers the lumberjack levels up while chopping trees.\n Similar to the builder, with higher levels their speed increases. The higher the hut level is the quicker they level up. \n However, their level is capped by the level of their home. Thus, if they are unhappy with their living conditions they will stop leveling until you provide them with better housing.",
+  "com.minecolonies.coremod.info.lumberjack.3": "As all workers do, Lumberjacks level up while chopping trees.\n At higher levels, the Lumberjack's speed increases. The higher their hut level and Intelligence skill, the quicker they level up. \n However, their level is capped by the level of their home. Thus, if they are unhappy with their living conditions, they will stop leveling unless you provide them with better housing.",
```

Comment:
This statement is technically false. Not all workers level up while chopping trees!